tex/measure/martingale.tex

```diff
@@ -759,10 +759,12 @@ \subsection{The ballot problem}
 	check it. But if you don't\dots
 \end{remark}
 
-\subsection{ABRACADABRA}
+\subsection{ABRACADABRA\texorpdfstring{\protect\footnote{%
+\url{https://www.jeremykun.com/2014/03/03/martingales-and-the-optional-stopping-theorem/}}}{}}
 
 
-\subsection{USA TST 2018}
+\subsection{USA TST 2018\texorpdfstring{\protect\footnote{%
+\url{https://aops.com/community/p9513099}}}{}}
```

Comment:
A sufficiently interested reader can lookup the source themselves. (Assume one even exist)

Not like I know why `\footnote` is fragile though.